e2e-tests/tests/users/04-update-default-user-password.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 10
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+       kubectl -n ${NAMESPACE} patch secret users-pguser-postgres --type merge --patch '{"stringData": {"password": "default-db-password", "verifier":""}}' \
+        || true
```

Comment:
Do we need it?

Reply:
fixed in 32bd532f903d269ccdb36d71fc60b3c28f12dd22